src/Gui/InputHint.cpp

```diff
@@ -0,0 +1,313 @@
+        case KeySlash: return "/";
+        case Key0: return "0";
+        case Key1: return "1";
+        case Key2: return "2";
+        case Key3: return "3";
+        case Key4: return "4";
+        case Key5: return "5";
+        case Key6: return "6";
+        case Key7: return "7";
+        case Key8: return "8";
+        case Key9: return "9";
+        case KeyColon: return ":";
+        case KeySemicolon: return ";";
+        case KeyLess: return "<";
+        case KeyEqual: return "=";
+        case KeyGreater: return ">";
+        case KeyQuestion: return "?";
+        case KeyAt: return "@";
+        case KeyA: return "A";
+        case KeyB: return "B";
+        case KeyC: return "C";
+        case KeyD: return "D";
+        case KeyE: return "E";
+        case KeyF: return "F";
+        case KeyG: return "G";
+        case KeyH: return "H";
+        case KeyI: return "I";
+        case KeyJ: return "J";
+        case KeyK: return "K";
+        case KeyL: return "L";
+        case KeyM: return "M";
+        case KeyN: return "N";
+        case KeyO: return "O";
+        case KeyP: return "P";
+        case KeyQ: return "Q";
+        case KeyR: return "R";
+        case KeyS: return "S";
+        case KeyT: return "T";
+        case KeyU: return "U";
+        case KeyV: return "V";
+        case KeyW: return "W";
+        case KeyX: return "X";
+        case KeyY: return "Y";
+        case KeyZ: return "Z";
+        case KeyBracketLeft: return "[";
+        case KeyBackslash: return "\\";
+        case KeyBracketRight: return "]";
+        case KeyUnderscore: return "_";
+        case KeyQuoteLeft: return "\"";
+        case KeyBraceLeft: return "{";
+        case KeyBar: return "Bar";
+        case KeyBraceRight: return "}";
+        case KeyAsciiTilde: return "~";
+
+        // misc keys
+        case KeyEscape: return "Escape";
+        case KeyTab: return "tab ⭾";
+        case KeyBacktab: return "Backtab";
+        case KeyBackspace: return "⌫";
+        case KeyReturn: return "Return";
+        case KeyEnter: return "Enter";
+        case KeyInsert: return "Insert";
+        case KeyDelete: return "Delete";
+        case KeyPause: return "Pause";
+        case KeyPrintScr: return "Print";
+        case KeySysReq: return "SysReq";
+        case KeyClear: return "Clear";
+
+        // cursor movement
+        case KeyHome: return "Home";
+        case KeyEnd: return "End";
+        case KeyLeft: return "⇦";
+        case KeyUp: return "⇧";
+        case KeyRight: return "⇨";
+        case KeyDown: return "⇩";
+        case KeyPageUp: return "PgDown";
+        case KeyPageDown: return "PgUp";
+
+        // modifiers
+        case KeyShift: return "Shift";
+#ifdef FC_OS_MACOSX
+        case KeyControl: return "⌘";
+#else
+        case KeyControl: return "Ctrl";
+#endif
+        case KeyMeta: return "Meta";
+#ifdef FC_OS_MACOSX
+        case KeyAlt: return "⌥";
+#else
+        case KeyAlt: return "Alt";
+#endif
```

Comment:
Meta needs is ctrl on mac (due to the switch ctrl->cmd).
That leaves us with just shift without a symbol, so adding that to the mix as well:
```cpp
#ifdef FC_OS_MACOSX
        case KeyShift: return "⇧";
        case KeyControl: return "⌘";
        case KeyMeta: return "⌃";
        case KeyAlt: return "⌥";
#else
        case KeyShift: return "Shift";
        case KeyControl: return "Ctrl";
        case KeyMeta: return "Meta";
        case KeyAlt: return "Alt";
#endif
```

The glyphs for shift and ctrl was copy-pasted from: https://gist.github.com/jlyonsmith/6992156f18c423fd1c5af068aa311fb5

I don't think we need to include all of them at this point as people would understand "Caps Lock". This could be future work.